app/views/claims/schools/show.html.erb

```diff
@@ -13,9 +13,9 @@
 
 <% content_for(:before_content) do %>
   <%= render PrimaryNavigationComponent.new do |component| %>
-    <% component.with_navigation_item t(".primary_navigation.claims"), "#" %>
-    <% component.with_navigation_item t(".primary_navigation.users"), "#" %>
-    <% component.with_navigation_item t(".primary_navigation.details"), claims_school_path(@school) %>
+    <% component.with_navigation_item t("navigation_bar.claims"), "#" %>
+    <% component.with_navigation_item t("navigation_bar.users"), "#" %>
+    <% component.with_navigation_item t("navigation_bar.details"), claims_school_path(@school) %>
```

Comment:
I have some concerns about this change. The same as here: https://github.com/DFE-Digital/itt-mentor-services/pull/112#issuecomment-1892254360

As the `PrimaryNavigationComponent` here and in the application layout are different, I'd suggest to separate their locale locations.

Reply:
Honestly, I was just pointing the failures to the thing that actually existed. `navigation_bar` exists and `primary_navigation` doesn't.

Reply:
I've removed the navigation bar translations as well as some other first level translations.